roles/os10_users/templates/os10_users.j2

```diff
@@ -21,6 +25,9 @@ username {{ item.username }} password {{ item.password }} role {{ item.role }}
 username {{ item.username }} password {{ item.password }}
           {% endif %}
         {% endif %}
+        {% if item.sshkey is defined and item.sshkey %}
+username {{ item.username }} sshkey "{{ item.sshkey }}" 
+        {% endif %}
```

Comment:
Please add the 'no' form here when sshkey is defined by None.

Reply:
added